bootstrap/app.php

```diff
@@ -19,8 +19,34 @@
     ->withEvents(discover: [
         __DIR__.'/../src/Shared/App/Listeners',
     ])
-    ->withMiddleware(callback: function (Middleware $middleware) {
-        //
+    ->withMiddleware(function (Middleware $middleware) {
+        $middleware->use([
+            \Illuminate\Foundation\Http\Middleware\InvokeDeferredCallbacks::class,
+            // Uncomment the following line if you are running your server from behind an HTTP proxy, such as a load balancer.
+            // \Illuminate\Http\Middleware\TrustProxies::class,
+            \Illuminate\Http\Middleware\TrustHosts::class, // Default: allSubdomainsOfApplicationUrl
```

Comment:
Would we want TrustHosts always or just in production? Does it matter or is the middleware already considering development?

Reply:
Why wouldn't we want to use it in development?